pom.xml

```diff
@@ -33,6 +33,16 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```

Comment:
I think we don't need to include any of these dependencies here. cos elide-standalone dependecy will pull everything. Please remove and verify,